fix(curriculum): use implicit return in callback function (#59327)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-advanced-array-methods-by-building-a-statistics-calculator/63507e4562cdde3a28e8de1b.md

> Add a callback function to your `.filter()` method that returns `true` if the element is not `NaN`.
> 
> Use an implicit return - you used implicit return in the `numbers` variable.

Reviewer: ```suggestion
Add a callback function to your `.filter()` method with `el` as its parameter, and use an implicit return to return `true` if the element is NOT `NaN`.
```

This is a little more concise. Users should know what an implicit return it from the music player project earlier.
